src/Stratis.Bitcoin/Builder/FullNodeBuilder.cs

```diff
@@ -197,9 +199,22 @@ public IFullNode Build()
                 fullNodeServiceProvider,
                 this.Features.FeatureRegistrations.Select(s => s.FeatureType).ToList()));
 
+            this.WritePID();
+
             return fullNode;
         }
 
+        /// <summary>
+        /// Writes a file called pid to DataDir containing the Process ID
+        /// </summary>
+        private void WritePID()
```

Comment:
This can be done in many different ways, I prefer not to do it in the node builder.
It can be done as an additional new feature or at the base feature level, or even one of the existing features.

What is the purpose of it and is it used but test only?